### ISSUE TITLE
Minify production build

### ISSUE THREAD

**Issue by kevinmeyvaert** [MakeInBelgium/babbelbox#42]

**Comment by vdwijngaert** [comment]
Yes, zeker een optie. Momenteel is er bewust geen development workflow (bundling/..) om de participatiedrempel zo laag mogelijk te houden.

Misschien herzien in de toekomst?

**Comment by toonvandeputte** [comment]
Het zit nu in Jekyll, dus kunnen we evt wel doen.
